models/quality_measures/intermediate/nqf0053_osteoporosis_management_in_women_who_had_a_fracture/quality_measures__int_nqf0053_numerator.sql

```diff
@@ -0,0 +1,270 @@
+        , 'bone mineral density tests cpt'
+        , 'bone mineral density tests icd10pcs'
+        , 'bone mineral density tests hcpcs'
+        , 'dexa dual energy xray absorptiometry, bone density'
+        , 'central dual energy x-ray absorptiometry (dxa)'
+        , 'spinal densitometry x-ray' 
+        , 'ultrasonography for densitometry' 
+        , 'ct bone density axial'
+        , 'peripheral dual-energy x-ray absorptiometry (dxa)'
+    )
+
+)
+
+, procedures_osteo_related as (
+
+    select
+        patient_id
+      , encounter_id
+      , procedure_date
+      , source_code_type
+      , source_code
+    from {{ref('quality_measures__stg_core__procedure')}} as procs
+    inner join osteo_procedure_codes
+        on procs.source_code = osteo_procedure_codes.code
+        and procs.source_code_type = osteo_procedure_codes.code_system
+
+)
+
+, qualifying_procedures as (
+
+    select procedures_osteo_related.*
+        , denominator.measure_id
+        , denominator.measure_name
+        , denominator.measure_version
+        , denominator.recorded_date
+        , denominator.performance_period_begin
+        , denominator.performance_period_end
+    from procedures_osteo_related
+    inner join denominator
+        on procedures_osteo_related.patient_id = denominator.patient_id
+        and 
+            procedures_osteo_related.procedure_date between
+            denominator.recorded_date 
+            and
+            {{dbt.dateadd (
+                      datepart = "month"
+                    , interval = +6
+                    , from_date_or_timestamp = "denominator.recorded_date"
+                          )
+            }} 
+
+)
+
+, denominator_patients_disqualified_from_procedure as (
+    
+    select 
+          denominator.patient_id
+        , denominator.measure_id
+        , denominator.measure_name
+        , denominator.measure_version
+        , denominator.recorded_date
+        , denominator.performance_period_begin
+        , denominator.performance_period_end  
+    from denominator
+    left join qualifying_procedures 
+    on denominator.patient_id = qualifying_procedures.patient_id
+    where qualifying_procedures.patient_id is null
+
+)
+
+-- pharmacy_claim begin
+
+, osteo_rx_codes as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from value_sets
+    where lower(concept_name) 
+        in 
+        ( 
+          'osteoporosis medications for urology care'
+        , 'osteoporosis medication'
+        , 'bisphosphonates'
+        )
+
+)
+
+, pharmacy_claims_osteo_related as (
+
+    select
+        patient_id
+      , dispensing_date
+      , ndc_code  
+    from {{ref('quality_measures__stg_pharmacy_claim')}} as pharmacy_claims
+    inner join osteo_rx_codes
+    on pharmacy_claims.ndc_code = osteo_rx_codes.code
+        and lower(osteo_rx_codes.code_system) = 'ndc'
+
+)
+
+, qualifying_pharmacy_claims as (
+
+    select pharmacy_claims_osteo_related.*
```

Comment:
Same note as above. 

> Can you list the columns from this table rather than using `*`? It would make the code much easier to read if users didn't have to keep going back to the original CTE to see what's included in the select.